### PR TITLE
[relnotes][skip-ci] Mention that TTree now supports long/unsigned long

### PR DESCRIPTION
**Pull request by eguiraud**

### CODE REVIEW

**Comment by eguiraud** [comment]
@Axel-Naumann good to go?

**Comment by Axel-Naumann** [comment]
@pcanal what's your thought about mentioning `size_t` here? It remains platform-dependent, but does that matter, actually?

File: README/ReleaseNotes/v624/index.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ See the discussion at [ROOT-11014](https://sft.its.cern.ch/jira/browse/ROOT-1101
 
 ## TTree Libraries
 
+- TTree now supports leaves of types `long` and `unsigned long` (and therefore also `std::size_t` on most systems). The corresponding leaflist letters are 'G' and 'g'.
```
Comment (pcanal) [inline README/ReleaseNotes/v624/index.md:69]:
```suggestion
- TTree now supports for branch in "leave list mode" leaves of types `long` and `unsigned long` (and therefore also `std::size_t` on most systems). The corresponding leaflist letters are 'G' and 'g'.
```

Comment (eguiraud) [reply README/ReleaseNotes/v624/index.md:69]: Hi @pcanal , thank you for taking a look. Can you please amend your suggestion with `s/for branch in "leave list mode"/, for branches in "leaflist mode",/`? :smile:

**Comment by Axel-Naumann** [comment]
@pcanal ping on this one...

**Review by pcanal — APPROVED** [review]

**Pull request merged**